src/core/api.js

```diff
@@ -8,17 +8,19 @@ export function getPopularMemes() {
 // request to create a meme at https://api.imgflip.com/caption_image
 export function createMeme(templateID, upperText, lowerText) {
   return request('caption_image', {
-    templateID,
-    upperText,
-    lowerText,  
+    template_id: templateID,
+    text0: upperText,
```

Comment:
The API needs to receive a `text0` and `text1` variables? :/

Reply:
Yes, unfortunately, it does. Nothing I can do about that 😕 https://api.imgflip.com/